Model/Payment/Method/Bread.php

```diff
@@ -375,11 +375,29 @@ public function canVoid() {
     public function capture(\Magento\Payment\Model\InfoInterface $payment, $amount)
     {
         if ($this->adminOrderHelper->isAdminOrder()) {
-            return; // Bypass logic for admin order creation
+            return $this; // Bypass logic for admin order creation
         }
         if (!$this->canCapture()) {
             throw new \Magento\Framework\Exception\LocalizedException(__('Capture action is not available.'));
         }
+
+        $transactionAdditionalInfo = $payment->getTransactionAdditionalInfo();
+
+        $settleResult = isset($transactionAdditionalInfo['settle_result']) 
```

Comment:
Shopify checks this value for all transaction updates.  Settle/Refund/Void/Capture/Authorize.  If the transaction is already in that state, we log and don't do anything.  Might make sense to do that for the others as well.

Reply:
@dschoder-bread good call. I'll update it

Reply:
@dschoder-bread done